### PR TITLE
Update event card section of Brigade profile page

### PR DESCRIPTION
**Pull request by tdooner**
- Add card samples to the style guide
- Re-introduce the "card--horizontal" variant for the "event card"
section of the brigade profile
- Remove unused classes from buttons

### CODE REVIEW

**Comment by tdooner** [comment]
Still plenty of work to do here but wanted to commit everything I've got before the weekend! 
image

File: brigade/static/scss/molecules/_card.scss [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
 // Card
 //
 
 // TODO: upstream this into the v4 style guide
+// TODO: Remove <a> selectors from all these cards
+a.card--horizontal {
+  align-items: center;
+  display: flex;
+
+  .card-image-container {
```
Comment (fritzjooste) [inline brigade/static/scss/molecules/_card.scss]: I'm guessing this is inherited from the v4 style guide, but eventually we can have a more-BEM class name like `.card__image`.

**Comment by tdooner** [comment]
Great, thanks for looking. Mind taking another look?

Here's how it behaves now:
card-responsiveness

**Comment by fritzjooste** [comment]
This is looking good! Some final suggested tweaks:
- make `card-content` and `card-image-container` 100% wide of mobile
- can we make the map image go full-width on mobile?

**Comment by tdooner** [comment]
Great! Got it. It looks good.

card-responsiveness

**Comment by fritzjooste** [comment]
That's awesome! I think it's good to go! :shipit:

-- 

Fritz Jooste // Frontend Developer, Code for America

**Pull request merged**